docs/openapi3.yaml

```diff
@@ -1365,10 +1411,51 @@ components:
         - dsl
         - basic
         example: dsl
+    bench_mark_response:
```

Comment:
we didn't actually create new 'test_id'
it's the same test
so the response can be the same as request and use one object

Reply:
fixed